iothub/device/src/InternalClient.cs

```diff
@@ -465,7 +465,7 @@ public async Task CompleteMessageAsync(string lockToken, CancellationToken cance
 
             try
             {
-                await InnerHandler.CompleteAsync(lockToken, cancellationToken).ConfigureAwait(false);
+                await InnerHandler.CompleteMessageAsync(lockToken, cancellationToken).ConfigureAwait(false);
```

Comment:
I renamed this in the clients, but missed the handlers. Now C2D methods make more sense as being related to messaging.